### PR TITLE
Documentation updates for curations related to new cpe property

### PR DESCRIPTION
**Pull request by oheger-bosch**
Refer to the individual commits for further details.

### CODE REVIEW

**Review by MarcelBochtler — APPROVED** [review]

File: integrations/schemas/repository-configuration-schema.json [hunk 1]
```diff
@@ -158,6 +161,9 @@
                   "homepage_url": {
                     "type": "string"
                   },
+                  "purl": {
+                    "type": "string"
+                  },
```
Comment (sschuberth) [inline integrations/schemas/repository-configuration-schema.json:166]: @MarcelBochtler how could this be missing if the file was auto-generated?

**Comment by oheger-bosch** [comment]
@tsteenbe As discussed in the last ORT developer's meeting, here is the PR that updates the documentation related to curations for the new `cpe` property.

File: integrations/schemas/repository-configuration-schema.json [hunk 2]
```diff
@@ -149,6 +149,9 @@
                   "concluded_license": {
                     "type": "string"
                   },
+                  "cpe": {
+                    "type": "string"
+                  },
```
Comment (sschuberth) [inline integrations/schemas/repository-configuration-schema.json:154]: Out of interest: What's the order of properties in this file? It seems to neither be alphabetical, nor matching the order in Kotlin source code.

**Review by sschuberth — DISMISSED** [review]

File: integrations/schemas/repository-configuration-schema.json (hunk 1 above)
Comment (MarcelBochtler) [reply integrations/schemas/repository-configuration-schema.json:166]: It was mostly auto-generated.
I had to remove some values and I guess `purl` was removed by accident.

File: integrations/schemas/repository-configuration-schema.json (hunk 2 above)
Comment (oheger-bosch) [reply integrations/schemas/repository-configuration-schema.json:154]: I was asking myself the same question. The first properties are sorted alphabetically, so I tried to adhere to this convention; but then the order gets mixed up.

**Review by sschuberth — DISMISSED** [review]

**Review by sschuberth — APPROVED** [review]

**Pull request merged**